modules/MultiDRM/MultiDRM.loader.js

```diff
@@ -140,7 +140,7 @@
 			var hlsSource = sources.filter( function ( source ) {
 				return ( source.mimeType === "application/vnd.apple.mpegurl" );
 			});
-			drmSources.push(hlsSource);
+			drmSources.concat(hlsSource);
 		}
```

Comment:
No, but filter returns an array so in order to merge both we require concat and not push.

Sent from Outlook Mobile<https://aka.ms/qtex0l>




On Tue, Apr 19, 2016 at 9:49 PM -0700, "Amir Chervinsky" <notifications@github.com<mailto:notifications@github.com>> wrote:


In modules/MultiDRM/MultiDRM.loader.js<https://github.com/kaltura/mwEmbed/pull/2702#discussion_r60350159>:

> @@ -140,7 +140,7 @@
>                       var hlsSource = sources.filter( function ( source ) {
>                               return ( source.mimeType === "application/vnd.apple.mpegurl" );
>                       });
> -                     drmSources.push(hlsSource);
> +                     drmSources.concat(hlsSource);
>               }


@OrenMe<https://github.com/OrenMe> - so hlsSource might contain more than one source? In that case, maybe call it hlsSources. but feel free to merge if you don't feel like changing the variable name...

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/kaltura/mwEmbed/pull/2702/files/d357f40c61fbd4e864ef2429ace5da008277f557#r60350159>